src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -94,7 +94,7 @@
   <ngx-datatable #table
                  class="bootstrap cd-datatable"
                  [cssClasses]="paginationClasses"
-                 [selectionType]="selectionType"
+                 [selectionType]="'checkbox'"
```

Comment:
This will change the selection type for ALL datatables in the whole UI to checkbox. Please remove.